.github/workflows/pr.yml

```diff
@@ -0,0 +1,18 @@
+name: Ava Tests on PR
+on: pull_request
```

Comment:
i simply made a copy of `main.yml` and changed these 2 lines

Reply:
i think we may be able to reuse the first one and just add a comma and an array?

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#using-multiple-events

Reply:
yup, confirmed: https://github.com/hchiam/test-blingblingjs/pull/2